CMakeLists.txt

```diff
@@ -975,9 +959,9 @@ if(MZ_BUILD_FUZZ_TESTS)
         set_target_properties(${target} PROPERTIES LINKER_LANGUAGE CXX)
         target_compile_definitions(${target} PRIVATE ${STDLIB_DEF})
         target_include_directories(${target} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
-        target_link_libraries(${target} ${PROJECT_NAME})
+        target_link_libraries(${target} ${MZ_TARGET})
 
-        add_dependencies(${target} ${PROJECT_NAME})
+        add_dependencies(${target} ${MZ_TARGET})
```

Comment:
isn't dependency already guaranteed after previous call to target_link_libraries?

Reply:
Indeed it's useless, but I prefer to not remove it in this PR to reduce reviewing complexity.